<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DirectoriesDialog</class>
 <widget class="QDialog" name="DirectoriesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Public key and password entry directories</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="infoLabel">
       <property name="text">
        <string>Info</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="defaultDirCheckbox">
       <property name="text">
        <string>Save private key in SherPass configuration directory</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Public key directory</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="pubkeyDirEdit"/>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pubkeyBrowseButton">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Password info directory</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="passDirEdit"/>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="passBrowseButton">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="privkeyDirLabel">
         <property name="text">
          <string>Private key directory</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="privkeyDirEdit"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="privkeyBrowseButton">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>38</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>defaultDirCheckbox</tabstop>
  <tabstop>privkeyDirEdit</tabstop>
  <tabstop>privkeyBrowseButton</tabstop>
  <tabstop>pubkeyDirEdit</tabstop>
  <tabstop>pubkeyBrowseButton</tabstop>
  <tabstop>passDirEdit</tabstop>
  <tabstop>passBrowseButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DirectoriesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DirectoriesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
